sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/ContainerInstanceManagementClient.cs

```diff
@@ -343,11 +336,10 @@ private void Initialize()
         {
             ContainerGroups = new ContainerGroupsOperations(this);
             Operations = new Operations(this);
-            ContainerGroupUsage = new ContainerGroupUsageOperations(this);
```

Comment:
It is breaking change to remove operations. Was it intentional?

Reply:
Yes, in the spec there were a few operations that were under different operation groups and i consolidated them because it made more sense. So there was the usage one you mentioned and a few others like cached images and capabilities that are all operations for a specific location so i grouped them all under the location operations

Reply:
I have put the link to the spec and PR for the spec in the description of this PR